src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/taglets/LinkTaglet.java

```diff
@@ -249,7 +249,8 @@ Content linkSeeReferenceOutput(Element holder,
                 }
             }
             if (refSignature.trim().startsWith("#") &&
-                    ! (utils.isPublic(containing) || utils.isLinkable(containing))) {
+                    ! (utils.isPublic(containing) || utils.isLinkable(containing)) &&
+                    ! (utils.isPrivate(refMem) ||utils.isPackagePrivate(refMem))) {
```

Comment:
Shouldn't this take into account whether we are running with -private or -package options? Ideally using utils.isIncluded(Element) would take care of this and give the right answer in all cases, although I haven't tried.

Reply:
I observed no changes when building javadoc using `-private` or `-package` before/after the change, I don't think this code path is taken in those cases.
`utils.isIncluded(refMem)`/`utils.isIncluded(containing)` is always false, I wasn't sure how to use it. (Which element should I check?)